allennlp/service/predictors/wikitables_parser.py

```diff
@@ -78,9 +81,9 @@ def _execute_logical_form_on_table(logical_form, table):
         with open(logical_form_filename, 'w') as temp_file:
             temp_file.write(logical_form + '\n')
 
-        table_dir = os.path.join(SEMPRE_DIR, 'csv/')
+        table_dir = os.path.join(SEMPRE_DIR, 'tsv/')
```

Comment:
Why did this crash before?  Seems odd that this would make a difference.

Reply:
The problem was the file had suffix ".csv" when it was really written in tsv format. This meant sempre was reading it incorrectly. I also changed the directory name to "tsv/" since it is more appropriate for its contents.

Reply:
Oh, I see, sempre was trying to be smart about the file name.  That's good to know.  Can you add a comment here that using `.tsv` is important?  Also, you can merge with the current wikitables branch, or rebase onto it, in order to get the tests to pass.

Reply:
done